.github/workflows/ci-lint-test.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         shell: bash
     strategy:
       matrix:
-        python: ["3.8", "3.10", "3.12"]  # 5 versions / years of coverage
+        python: ["3.8", "3.9", "3.10", "3.11", "3.12"]
```

Comment:
that fine granularity is not really needed and only delay test-results. The note was meant as semi-rule for future maintainers.

Reply:
> that fine granularity is not really needed and only delay test-results. The note was meant as semi-rule for future maintainers.

I can put the note back, but comprehensive testing is important. The tests run in parallel, so results don't really get delayed, and things change between versions. It's important to make sure this works for all target versions.

Reply:
Yes things change, but if 3.8 and 3.12 are passing, the versions inbetween will also work, as there are only new features or removal of deprecated modules. No major behavioral changes inbetween.

Reply:
but ok.